mutag_calib/configs/params/plotting_style.yaml

```diff
@@ -8,24 +8,31 @@ plotting_style:
       - TTto4Q__TTto4Q_l
       - SingleTop__SingleTop_l
       - VJets__VJets_l
+      - SingleTop__SingleTop_l
     "c":
       - QCD_MuEnriched__QCD_MuEnriched_c
       - TTto4Q__TTto4Q_c
       - SingleTop__SingleTop_c
       - VJets__VJets_c
+      - SingleTop__SingleTop_c
       - QCD_MuEnriched__QCD_MuEnriched_cc
       - TTto4Q__TTto4Q_cc
       - SingleTop__SingleTop_cc
       - VJets__VJets_cc
+      - SingleTop__SingleTop_cc
     "b":
       - QCD_MuEnriched__QCD_MuEnriched_b
       - TTto4Q__TTto4Q_b
       - SingleTop__SingleTop_b
       - VJets__VJets_b
+      - SingleTop__SingleTop_b
```

Comment:
I think in these dictionaries there are duplicate entries for SingleTop subsamples. Could you remove the duplicates?